spel/minimal-linux.json

```diff
@@ -319,6 +341,17 @@
             ],
             "type": "shell"
         },
+        {
+            "execute_command": "{{ .Vars }} sudo -E /bin/sh -ex '{{ .Path }}'",
+            "inline": [
+                "echo Unmounting /mnt",
+                "test $(grep -c /mnt /proc/mounts) -eq 0 || umount /mnt"
```

Comment:
I'm guessing you could collapse this block with the prior one. It shouldn't hurt the other builders to add the `test ... || umount ...` there.

Reply:
Done (https://github.com/plus3it/spel/pull/439/commits/bced277e56bfcceede160cfa33c4ddab17c67cd7).  Testing all these changes now.  (Should be about 30 mins; my dev system's not fast).

Reply:
Tested successfully.  I thought separate commits might be easier to (re)review, but if you want me to squash them all into a single commit, just let me know.